index.js

```diff
@@ -90,24 +90,42 @@ class ServerlessWebpack {
       'after:deploy:createDeploymentArtifacts': () => BbPromise.bind(this)
         .then(this.cleanup),
 
-      'before:deploy:function:packageFunction': () => BbPromise.bind(this)
-        .then(this.validate)
-        .then(this.compile)
-        .then(this.packExternalModules)
-        .then(this.packageModules),
+      'before:deploy:function:packageFunction': () => {
+        if (this.options.functionObj.webpack === false) {
+          return;
```

Comment:
All hooks must return promises. We cannot guarantee that the caller is aware that a non-promise can be returned that cannot be awaited. As we are a plugin we must make no assumptions about our host.
```
return BbPromise.resolve();
```